Makefile

```diff
@@ -177,7 +177,7 @@ DEVICETREE_DTB_amd64=
 DEVICETREE_DTB_arm64=$(DIST)/dtb/eve.dtb
 DEVICETREE_DTB=$(DEVICETREE_DTB_$(ZARCH))
 
-CONF_FILES=$(shell ls -d $(CONF_DIR)/*)
+CONF_FILES=$(if $(strip $(REMOTE_ACCESS_DISABLED)),$(shell ls -d $(CONF_DIR)/*), $(shell ls -d $(CONF_DIR)/* | grep -v 'remote_access_disabled'))
```

Comment:
I don't think we want to have users build their own image to enable this.

Instead the in argument to pkg/eve/runme.sh can be used to specify a directory with an empty remote_access_disabled file which will then produce an installer_{raw,iso,net} (depending on command) with that file added to /config partition.